### PR TITLE
Fixed course being accessed from studio of another site

### PR DESCRIPTION
**Pull request by manan-memon**
This PR fixes the issue of course endpoints of a different organization being accessed from another site for admin. 

JIRA: https://edlyio.atlassian.net/browse/EDLY-7167

### CODE REVIEW

File: common/djangoapps/student/auth.py [hunk 1]
```diff
@@ -105,6 +105,19 @@ def get_user_permissions(user, course_key, org=None):
...
+
+    :param course_key: a CourseKey object representing the course
+    :return: True if the course organization is in the site organization filters, False otherwise
+    """
+    site_orgs = configuration_helpers.get_current_site_orgs()
+    if course_key.org not in site_orgs:
+        return False
+    return True
+
+
```
Comment (Anas-hameed) [inline common/djangoapps/student/auth.py:120]: Isn't this function duplicate? Can you define a single function in EDLY app and import from there?

**Review by Anas-hameed — CHANGES_REQUESTED** [review]

File: common/djangoapps/student/auth.py (hunk 1 above)
Comment (manan-memon) [reply common/djangoapps/student/auth.py]: As we discussed, I have changed it to use the `is_course_org_same_as_site_org` from edly app

File: common/djangoapps/student/auth.py [hunk 2]
```diff
@@ -118,6 +131,10 @@ def has_studio_write_access(user, course_key):
     :param user:
     :param course_key: a CourseKey
     """
+    # verify if the course is from requesting site
```
[the remark this answers is not in the document]
Comment (manan-memon) [reply common/djangoapps/student/auth.py]: Removed

**Review by Anas-hameed — APPROVED** [review]
LGTM

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Pull request merged**